cfme/tests/containers/test_manageiq_ansible_users.py

```diff
@@ -0,0 +1,138 @@
+
+        """
+    setup_ansible_script(provider, script='create_user',
+                         values_to_update=users_values_to_create, script_type='users')
+    run_ansible('create_user')
+    navigate_to(User, 'All')
+    names_list = []
+    projects_table = Table(table_el)
+    for row in projects_table.rows():
+        names_list.append(row.full_name.text)
+    assert users_values_to_create.get('fullname') in names_list
+
+
+@pytest.mark.polarion('CMP-10555')
+def test_manageiq_ansible_update_user(ansible_users, provider):
+    """This test checks updating a User using Ansible script via Manage IQ module
+        Steps:
+        1. 'update_user.yml script runs against the appliance and adds a new user
+        2. Test navigates to Settings page and verifies the user was updated under Access Control
+
+        """
+    setup_ansible_script(provider, script='update_user',
+                         values_to_update=users_values_to_update, script_type='users')
+    run_ansible('update_user')
+    pytest.sel.refresh()
+    navigate_to(User, 'All')
+    names_list = []
+    projects_table = Table(table_el)
+    for row in projects_table.rows():
+        names_list.append(row.full_name.text)
+    assert users_values_to_update.get('fullname') in names_list
+
+
+@pytest.mark.polarion('CMP-10556')
+def test_manageiq_ansible_create_same_name_user(ansible_users, provider):
+    """This test checks updating a User using Ansible script via Manage IQ module
+        Steps:
+        1. 'update_user.yml script runs against the appliance and adds a new user
+        2. Test navigates to Settings page and verifies the user was updated under Access Control
+
+        """
+    setup_ansible_script(provider, script='create_user',
+                         values_to_update=users_values_to_create, script_type='users')
+    run_ansible('create_user')
+    pytest.sel.refresh()
+    navigate_to(User, 'All')
+    names_list = []
+    projects_table = Table(table_el)
+    for row in projects_table.rows():
+        names_list.append(row.full_name.text)
+    assert users_values_to_create.get('fullname') in names_list
+
+
+@pytest.mark.polarion('CMP-10557')
+def test_manageiq_ansible_bad_user_name(ansible_users, provider):
+    """This test checks updating a User using Ansible script with a bad username
+        Steps:
+        1. 'create_user_bad_user_name.yml script runs against the appliance tries to add a new user
+        2. Test navigates to Settings page and verifies the user was not added under Access Control
+
+        """
+    setup_ansible_script(provider, script='create_user_bad_user_name',
+                         values_to_update=users_values_to_create, script_type='users')
+    run_status = run_ansible('create_user_bad_user_name')
+    assert 'Authentication failed' in run_status
+    pytest.sel.refresh()
+    navigate_to(User, 'All')
+    names_list = []
+    projects_table = Table(table_el)
+    for row in projects_table.rows():
```

Comment:
I verify an error when I run the ansible and then check in GUI that nothing was added